tests/js/panelsTests.js

```diff
@@ -12,6 +12,19 @@ https://github.com/gpii/universal/LICENSE.txt
     "use strict";
 
     fluid.registerNamespace("gpii.tests");
+    fluid.registerNamespace("gpii.tests.prefs.panel.utils");
```

Comment:
Yes, latter seems good to me. Also, we need only one of these "registerNamespace" calls, the longer one